src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.TestLibrary/System.Text.Json.TestLibrary.targets

```diff
@@ -0,0 +1,17 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\System.Text.Json.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="TestClasses.cs" />
+  </ItemGroup>
+
+  <Target Name="FixIncrementalCoreCompileWithAnalyzers" BeforeTargets="CoreCompile">
```

Comment:
(question) Do we still need this? It feels wrong, but I don't really understand what it does.

Reply:
It's present in all projects that consume the analyzer.  It's needed because CSC wasn't treating Analyzers as inputs to CoreCompile and was not rebuilding when only the analyzer / generator changed.  I don't think we should remove it as part of this PR, but we can test it and see if its no longer needed and remove all instances in a separate PR.  Here's the issue in Roslyn: https://github.com/dotnet/roslyn/issues/1753